src/index.html

```diff
@@ -7,7 +7,55 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="body">
+    <div class="stars">
+      <div class="stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars--1">
+        <div class="active stars__star"></div>
```

Comment:
You shouldn't add class active to active stars, start should become active depending on modifier stars--n that you add to starts' wrapper